presto-hive/src/main/java/com/facebook/presto/hive/OrcFileWriterFactory.java

```diff
@@ -218,6 +219,7 @@ else if (com.facebook.hive.orc.OrcOutputFormat.class.getName().equals(storageFor
                     orcEncoding,
                     fileColumnNames,
                     fileColumnTypes,
+                    Optional.empty(),
```

Comment:
What is the reason for this additional parameter ?

Reply:
The diff looks good, except for this added parameter.

Reply:
Fixed it , not required.